web/index.html

```diff
@@ -195,6 +195,14 @@ <h3 class="header2 light"><small class="header">Can select multiple options</sma
                     </label>
                   </div>
                 </div>
+                <div class="col-md-12">
+                  <div class="checkbox">
+                    <label>
+                      <input type="checkbox" value="test,tests" id="tests" onClick="updateCheckbox(this)" />
```

Comment:
Should we also add the `rtpcr` here? 
```suggestion
                      <input type="checkbox" value="test,tests,rtpcr" id="tests" onClick="updateCheckbox(this)" />
```

Reply:
Interesting point @manassolanki. It's tempting to add the names of specific techniques like RTPCR (and variation RT-PCR), CBNAAT, etc. But @pshrimal21 mentioned that there are possible issues ("circuit breakers") with many OR'd keywords. Maybe add these specific techniques as separate entries/checkboxes?

Reply:
Yeah, we figure out Twitter circuit breakers a couple of days back while exploring the Twitter search functionality. I am getting better and relevant results in the case of using terms like `rtpcr` instead of `tests` since `test` is too broad and generic. But yeah, I might be wrong here.